### DISCUSSION TITLE
Configuration when the vite app is in an app dir

### DISCUSSION THREAD

**Discussion by ftimurs** [MrBin99/django-vite#108] [Q&A] [answered]
I can't figure out of one would configure the vite.config.js if the vite application lives inside on of the django apps. In my case I have multiple django apps, and each django app has at least one react app.

So, in my case, one of my Django apps would have its static folder on the same level as the vite folder, so the config file would need to refer one directory down (i.e. "../static/") but when i run `npm run build` it complains and doesn't use base.

**Comment by MrBin99** [comment, marked as answer]
If I unerstand, you have one vite.config.js in each app ?

I suggest you, if you can, to have one frontend pipeline for all your apps (one vite.config.js at the root).
It has many advantages : it's be simpler to configure, you can share dependencies between your React apps and the most important you can customize how vite will split the code, producing of file per apps, if this is the desired behaviour : https://vitejs.dev/guide/build.html#chunking-strategy
